README.md

```diff
@@ -291,6 +291,22 @@ Obtain SMTP credentials from a provider for transactional emails. Set the SMTP_U
 
 <hr>
 
+<img src="https://s.imgur.com/images/favicon-96x96.png" height="75">
+
+- Go to <a href="https://api.imgur.com/oauth2/addclient" target="_blank">https://api.imgur.com/oauth2/addclient</a>
+- Sign in or create an Imgur account
+- Fill out the registration form:
+  - **Application name**: Your app name
+  - **Authorization type**: Select "OAuth 2 authorization without a callback URL"
+  - **Application website**: Your BASE_URL value (i.e. `http://localhost:8080`)
+  - **Email**: Your email address
+  - **Description**: Brief description of your application
+- After registration, you'll receive a **Client ID** and **Client Secret**
+- Copy the **Client ID** and paste it into your `.env` file as `IMGUR_CLIENT_ID`
+- **Note**: For this API example, you only need the Client ID. The Client Secret is used for more advanced OAuth flows.
```

Comment:
I didn't realize they have an oauth integration.  Since we are trying to reduce the user efforts during a hackathon, can you use also do the oauth integration?  Take a look at the example for Twitch and GitHub.  They each have a section of the view that requires authentication, and the other part that doesn't.